test/controllers/property.controller.test.js

```diff
@@ -51,28 +90,235 @@ describe('Add Property Route', () => {
+          .post('/api/v1/property/add')
+          .set('authorization', adminToken)
+          .send(property)
+          .end((err, res) => {
+            expect(res).to.have.status(412);
+            expect(res.body.success).to.be.eql(false);
+            expect(res.body.message).to.be.eql('Validation Error');
+            expect(res.body.error).to.be.eql('"Toilet number" must be a number');
+            done();
+          });
+      });
+    });
+    context('when description is empty', () => {
+      it('returns an error', (done) => {
+        const property = PropertyFactory.build({ description: '' });
+        request()
+          .post('/api/v1/property/add')
+          .set('authorization', adminToken)
+          .send(property)
+          .end((err, res) => {
+            expect(res).to.have.status(412);
+            expect(res.body.success).to.be.eql(false);
+            expect(res.body.message).to.be.eql('Validation Error');
+            expect(res.body.error).to.be.eql('"Property description" is not allowed to be empty');
+            done();
+          });
+      });
+    });
+    context('when floorPlans is empty', () => {
+      it('returns an error', (done) => {
+        const property = PropertyFactory.build({ floorPlans: '' });
+        request()
+          .post('/api/v1/property/add')
+          .set('authorization', adminToken)
+          .send(property)
+          .end((err, res) => {
+            expect(res).to.have.status(412);
+            expect(res.body.success).to.be.eql(false);
+            expect(res.body.message).to.be.eql('Validation Error');
+            expect(res.body.error).to.be.eql('"Property floor plans" is not allowed to be empty');
+            done();
+          });
+      });
+    });
+    context('when neighborhood is empty', () => {
+      it('returns an error', (done) => {
+        const property = PropertyFactory.build({ neighborhood: '' });
+        request()
+          .post('/api/v1/property/add')
+          .set('authorization', adminToken)
+          .send(property)
+          .end((err, res) => {
+            expect(res).to.have.status(412);
+            expect(res.body.success).to.be.eql(false);
+            expect(res.body.message).to.be.eql('Validation Error');
+            expect(res.body.error).to.be.eql('"Property neighborhood" must be an array');
+            done();
+          });
+      });
+    });
+    context('when mainImage is empty', () => {
+      it('returns an error', (done) => {
+        const property = PropertyFactory.build({ mainImage: '' });
+        request()
+          .post('/api/v1/property/add')
+          .set('authorization', adminToken)
+          .send(property)
+          .end((err, res) => {
+            expect(res).to.have.status(412);
+            expect(res.body.success).to.be.eql(false);
+            expect(res.body.message).to.be.eql('Validation Error');
+            expect(res.body.error).to.be.eql('"Property mainImage" is not allowed to be empty');
```

Comment:
Based on the above nits:
```suggestion
            expect(res.body.error).to.be.eql('"Property main image" is not allowed to be empty');
```